python/ray/autoscaler/aws/example-multi-node-type.yaml

```diff
@@ -17,25 +17,29 @@ available_node_types:
         node_config:
             InstanceType: m4.xlarge
         resources: {"CPU": 4}
+        min_workers: 0
```

Comment:
Let's set this to 1 for this, as an example. And remove the zero entries for the rest.